spec/index.bs

```diff
@@ -1761,6 +1801,80 @@ enum GPUCompareFunction {
 };
 </script>
 
+<dl dfn-type="enum-value" dfn-for=GPUCompareFunction>
+    : <dfn>"never"</dfn>
+    ::
+        Comparison tests never pass.
+
+    : <dfn>"less"</dfn>
+    ::
+        A provided value passes the comparison test if it is less than the stored value.
+
+    : <dfn>"equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is equal to the stored value.
+
+    : <dfn>"less-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is less than or equal to the stored value.
+
+    : <dfn>"greater"</dfn>
+    ::
+        A provided value passes the comparison test if it is greater than the stored value.
+
+    : <dfn>"not-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is not equal to the stored value.
+
+    : <dfn>"greater-equal"</dfn>
+    ::
+        A provided value passes the comparison test if it is greater than or equal to the stored value.
+
+    : <dfn>"always"</dfn>
+    ::
+        Comparison tests always pass.
+</dl>
+
+<div algorithm>
+    <dfn abstract-op>validating GPUSamplerDescriptor</dfn>(device, descriptor)
+    **Arguments:**
+        - {{GPUDevice}} |device|
+        - {{GPUSamplerDescriptor}} |descriptor|
+    
+    **Returns:** boolean
+
+    Return true if and only if all of the following conditions apply:
+        - |device| is valid.
+        - |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}} is greater than or equal to 0.
+        - |descriptor|.{{GPUSamplerDescriptor/lodMaxClamp}} is greater than or equal to
+            |descriptor|.{{GPUSamplerDescriptor/lodMinClamp}}.
+</div>
+
+### {{GPUDevice}}.<dfn method for=GPUDevice>createSampler(descriptor)</dfn> ### {#sampler-createsampler}
+
+<div algorithm=GPUDevice.createSampler>
+    **Arguments:**
+        - optional {{GPUSamplerDescriptor}} |descriptor| = {}
+
+    **Returns:** {{GPUSampler}}
+
+    1. Let |s| be a new {{GPUSampler}} object.
+    1. If |descriptor| is null or undefined:
+        1. Set |s|.{{GPUSampler/[[descriptor]]}} to a new {{GPUSamplerDescriptor}} object with specified default attributes.
+    1. Else:
+        1. Set |s|.{{GPUSampler/[[descriptor]]}} to |descriptor|.
```

Comment:
I think we should adopt a convention where, here, `|descriptor|` is always defined and we don't need to write text to describe what happens. The rules for this are provided by WebIDL (since we have `= {}` above).
```suggestion
    1. Set |s|.{{GPUSampler/[[descriptor]]}} to |descriptor|.
```

Reply:
Fixed!